test/quantization/test_backward_compatibility.py

```diff
@@ -53,7 +54,7 @@ def remove_prefix(text, prefix):
             if input_quantized:
                 input_tensor = torch.quantize_per_tensor(input_tensor, 0.5, 2, torch.quint8)
             torch.save(input_tensor, input_file)
-            torch.save(qmodule.state_dict(), state_dict_file)
+            torch.save(qmodule.state_dict(), state_dict_file, _use_new_zipfile_serialization=new_zipfile_serialization)
```

Comment:
Can you add a comment stating this is temporary and point to the relevant PR that needs to land? If anyone is using LSTMs internally, they will need to use this hack to serialize their models.